src/main/java/hudson/plugins/release/dashboard/RecentReleasesPortlet.java

```diff
@@ -161,18 +164,33 @@ public Calendar getEntryTimestamp(Run entry) {
             return entry.getTimestamp();
         }
 
+        @Override
         public String getEntryAuthor(Run entry) {
-        	// release builds are manual so get the UserCause
-        	// and report rss entry as user who kicked off build
-        	List<Cause> causes = entry.getCauses();
-        	for (Cause cause : causes) {
-        		if (cause instanceof UserCause) {
-        			return User.get(((UserCause) cause).getUserName()).getFullName();
-        		}
-        	}
-        	
-        	// in the unexpected case where there is no user cause, return admin
-            return Mailer.descriptor().getAdminAddress();
+            // release builds are manual so get the UserCause
+            // and report rss entry as user who kicked off build
+            List<Cause> causes = entry.getCauses();
+            for (Cause cause : causes) {
+                final String userName;
+                if (cause instanceof UserIdCause) {
+                    userName = ((UserIdCause)cause).getUserName();
+                } else if (cause instanceof UserCause) {
+                    userName = ((UserCause)cause).getUserName();
+                } else {
+                    userName = null;
```

Comment:
Won't work unless the first cause is any of the two user causes, if the first cause is something else you won't be able to set it in the next loop around. Effectively you are only looking at the first cause in the list.